src/screens/Profile/FavoriteList/index.tsx

```diff
@@ -1,208 +1,89 @@
+import List from '../../../components/item-groups';
 import ThemeIcon from '../../../components/theme-icon';
-import ThemeText from '../../../components/text';
+import {useFavorites} from '../../../favorites/FavoritesContext';
+import {LocationFavorite} from '../../../favorites/types';
+import MessageBox from '../../../message-box';
+import {RootStackParamList} from '../../../navigation';
+import {StyleSheet, Theme} from '../../../theme';
+import BackHeader from '../BackHeader';
 
 export type ProfileScreenNavigationProp = StackNavigationProp<
   ProfileStackParams,
-  'ProfileHome'
->;
-
-type ProfileNearbyScreenNavigationProp = CompositeNavigationProp<
-  StackNavigationProp<TabNavigatorParams, 'Profile'>,
-  ProfileScreenNavigationProp
+  'FavoriteList'
 >;
 
 type ProfileScreenProps = {
-  navigation: ProfileNearbyScreenNavigationProp;
+  navigation: StackNavigationProp<RootStackParamList>;
 };
 
-export default function Profile({navigation}: ProfileScreenProps) {
-  const css = useProfileStyle();
+export default function FavoriteList({navigation}: ProfileScreenProps) {
+  const style = useProfileStyle();
   const {favorites} = useFavorites();
   const items = favorites ?? [];
 
   const navigateToEdit = (item: LocationFavorite) => {
     navigation.navigate('AddEditFavorite', {editItem: item});
   };
-  const navigateToNearby = (item: LocationFavorite) => {
-    navigation.navigate('Nearest', {
-      location: {
-        ...item.location,
-        resultType: 'favorite',
-        favoriteId: item.id,
-      },
-    });
-  };
-
   const onAddButtonClick = () => navigation.push('AddEditFavorite', {});
-
-  const navigateHome = useNavigateHome();
-  const chatIcon = useChatIcon();
+  const onSortClick = () => navigation.push('SortableFavoriteList');
 
   return (
-    <SafeAreaView style={css.container}>
-      <Header
-        title="Mitt AtB"
-        leftButton={{
-          icon: <ThemeIcon svg={LogoOutline} />,
-          onPress: navigateHome,
-          accessibilityLabel: 'Gå til startskjerm',
-        }}
-        rightButton={chatIcon}
-      />
+    <SafeAreaView style={style.container}>
+      <BackHeader title="Favorittsteder" />
 
       <ScrollView>
-        <EditableListGroup
-          title="Favoritter"
-          data={items}
-          renderItem={(item) => (
-            <Item
-              item={item}
-              onEdit={() => navigateToEdit(item)}
-              onClick={navigateToNearby}
+        {!items?.length && (
+          <MessageBox
+            containerStyle={style.empty}
+            message="Du har ingen favorittsteder. Du kan legg til et nå ved å trykke på knappen under."
+            type="info"
+          />
+        )}
+
+        <List.Group withTopMargin>
+          {items.map((favorite) => (
+            <List.Favorite
+              key={favorite.name + favorite.location.id}
+              favorite={favorite}
+              onPress={() => navigateToEdit(favorite)}
             />
```

Comment:
Perhaps provide an accessibilitylabel/hint describing the onPress action? To help screen reader users understand what happens if they "activate" the favorite from here.